#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-gcc-4.7.dpatch by gregor herrmann <gregoa@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: add missing include to avoid FTBFS with gcc 4.7
## DP: Bug-Debian: http://bugs.debian.org/667238

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' lfhex-0.42~/src/expr.h lfhex-0.42/src/expr.h
--- lfhex-0.42~/src/expr.h	2008-09-19 02:36:49.000000000 +0200
+++ lfhex-0.42/src/expr.h	2012-05-07 17:03:22.788794002 +0200
@@ -19,6 +19,7 @@
  */
 
 #include <string>
+#include <sys/types.h>
 
 bool expr_eval( const std::string &str, off_t &value );
 
